### DISCUSSION TITLE
AttributeError: module 'torch' has no attribute 'use_deterministic_algorithms'

### DISCUSSION THREAD

**Discussion by ataML** [FETS-AI/Challenge#129] [Q&A] [answered]
Hi, 

I get the error  AttributeError: module 'torch' has no attribute 'use_deterministic_algorithms' as i run the notebook. I  had run experiments before without this issue. I'm using the torch version 1.8 and python 3.8.

Could you please help me solve the problem?

Thanks, 
Ata

**Comment by sarthakpati** [comment, marked as answer]
Hi,

One of these 2 solutions found on stack overflow could be useful:
- Restart Jupyter notebook kernel [[ref](https://stackoverflow.com/a/67472246/1228757)]
- Delete the virtual environment and re-initialize it again with a fresh install of the latest PyTorch version [[ref](https://stackoverflow.com/a/60822529/1228757)]

**Reply by ataML** [reply]
Thanks for the reply, after struggling  with setting the virtual environment and installing the latest version of  pytorch it finally worked

**Reply by sarthakpati** [reply]
Glad to hear that! Out of curiosity, have you submitted your algorithm for leaderboard-2 ranking?

**Reply by ataML** [reply]
sorry for the late reply, didnt get the notification for it. No I did not submit the algorithm, but used the challenge data, and code for my thesis.
